editor/plugins/mesh_instance_3d_editor_plugin.h

```diff
@@ -43,7 +43,10 @@ class MeshInstance3DEditor : public Control {
 	GDCLASS(MeshInstance3DEditor, Control);
 
 	enum Menu {
-		MENU_OPTION_CREATE_STATIC_TRIMESH_BODY,
+		MENU_OPTION_CREATE_TRIMESH_STATIC_BODY,
+		MENU_OPTION_CREATE_SINGLE_CONVEX_STATIC_BODY,
+		MENU_OPTION_CREATE_SIMPLIFIED_CONVEX_STATIC_BODY,
+		MENU_OPTION_CREATE_MULTIPLE_CONVEX_CSTATIC_BODY,
```

Comment:
```suggestion
		MENU_OPTION_CREATE_MULTIPLE_CONVEX_STATIC_BODY,
```
Assuming this is a typo